pkg/controllers/edgeconnect/controller.go

```diff
@@ -8,8 +8,8 @@ import (
 
 	"github.com/Dynatrace/dynatrace-operator/pkg/api/scheme"
 	"github.com/Dynatrace/dynatrace-operator/pkg/api/status"
-	edgeconnectv1alpha1 "github.com/Dynatrace/dynatrace-operator/pkg/api/v1alpha1/edgeconnect"
-	"github.com/Dynatrace/dynatrace-operator/pkg/clients/edgeconnect"
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1alpha1/edgeconnect"
+	ecclient "github.com/Dynatrace/dynatrace-operator/pkg/clients/edgeconnect"
```

Comment:
```suggestion
	edgeconnectClient "github.com/Dynatrace/dynatrace-operator/pkg/clients/edgeconnect"
```
To be consistent with other locations where e.g. "dynakubeComponents" was used in a similar situation.

Reply:
fixed fixed in 3f1de959